README.md

```diff
@@ -7,25 +7,24 @@
 Stream VR games from your PC to your headset via Wi-Fi.
 This is a fork of [ALVR](https://github.com/polygraphene/ALVR).
 
-### Direct download to the lastest version:
+### Direct download to the latest version:
 ### [Windows Launcher](https://github.com/alvr-org/ALVR/releases/latest/download/alvr_launcher_windows.zip) | [Linux Launcher](https://github.com/alvr-org/ALVR/releases/latest/download/alvr_launcher_linux.tar.gz)
 
 ## Compatibility
 
-|      VR Headset       |                                           Support                                            |
-| :-------------------: | :------------------------------------------------------------------------------------------: |
-|  Quest 1/2/3/3S/Pro   |     :heavy_check_mark: ([store link](https://www.meta.com/experiences/7674846229245715))     |
-| Pico Neo 3/4/4 Ultra  |                                      :heavy_check_mark:                                      |
-| Vive Focus 3/XR Elite |                                      :heavy_check_mark:                                      |
-|        YVR 1/2        |                                      :heavy_check_mark:                                      |
-|        Lynx R1        |                                      :heavy_check_mark:                                      |
-|   Apple Vision Pro    |       :heavy_check_mark: ([store link](https://apps.apple.com/app/alvr/id6479728026))        |
-| PhoneVR (smartphone)  | :heavy_check_mark: * (Check [PhoneVR](https://github.com/PhoneVR-Developers/PhoneVR) Client) |
-|    Android/Monado     |                                         :warning: *                                          |
-|        GearVR         |                                    :construction: (maybe)                                    |
-|       Oculus Go       |                                            :x: **                                            |
-
-\* : Only works on some smartphones, not enough testing.
+|          VR Headset          |                                       Support                                        |
+| :--------------------------: | :----------------------------------------------------------------------------------: |
+|       Apple Vision Pro       |   :heavy_check_mark: ([store link](https://apps.apple.com/app/alvr/id6479728026))    |
+|      Quest 1/2/3/3S/Pro      | :heavy_check_mark: ([store link](https://www.meta.com/experiences/7674846229245715)) |
```

Comment:
Add a little asterisk (yes another one) that you need to download it from the github releases for quest 1, because this has confused a couple people in past. Fixes #2532

Reply:
Should we also remove the mention to Oculus Go? So we just don't bother about porting patches to v18.2.3